app/models/learning_object.rb

```diff
@@ -82,7 +82,7 @@ def next(current_user,week_number) # mozno radsej spustit odporucanie.. pri tomt
     week = setup.weeks.find_by_number(week_number)
     RecommenderSystem::Recommender.setup(current_user.id,week.id,nil)
     best = RecommenderSystem::HybridRecommender.new.get_best(nil)
-    los = LearningObject.find(best[0])
+    los = LearningObject.find(best[0]) if best
```

Comment:
@veronikagondova @mlabaj 
Toto mi padalo v testoch. Mozem tu nechat tuto kontrolu, ze ak odporucac nic nevygeneruje, metoda vrati `nil` (standardne ak odporucac nic nevygeneruje, spadne to)?